### ISSUE TITLE
Custom SSL certificate

### ISSUE THREAD

**Issue by jdelaune** [localstack/localstack#5502]
### Is there an existing issue for this?

- [X] I have searched the existing issues and read the documentation

### Question

Back in 0.12 you could put a server.test.pem in the data directory and it would use that as a custom SSL certificate. That doesn’t seem to work in later versions (0.13/0.14). What is the correct way to do this now? I have tried hoofing it in `/var/lib/localstack/cache/` and that works but it seems to get overwritten every now and then. I have searched the documentation but couldn’t find anything on the matter. Thanks.

**Comment by thrau** [comment]
hey @jdelaune, there's currently no clean mechanism to do so, but your approach seems correct to me. localstack looks for `/var/lib/localstack/cache/server.test.pem`, but will overwrite it with a new version if that file has not been modified in the last 6 hours.
so essentially, localstack accepts your certificate, but treats it like a cached version of our cert, and `now() - mtime(<cert>)` is >= 6 hours, localstack will download and overwrite a new cert.

it's expected behavior, but i can see that this is not helpful for your usecase. we could introduce an environment variable that's lets you specify the path to the cert. would that work for your use case?

**Comment by jdelaune** [comment]
Yeah that would do it. I wasn't sure if `SKIP_SSL_CERT_DOWNLOAD` would stop the overwrite but I guess not?

**Comment by thrau** [comment]
i think you may have uncovered a problem with the configuration :-) it seems that `SKIP_SSL_CERT_DOWNLOAD` in combination with mounting a file to `/var/lib/localstack/cache/server.test.pem` should enable the behavior you want. but i think we may not be using `SKIP_SSL_CERT_DOWNLOAD` correctly, this is part of our pro codebase though.

i think the best way would be to make `SKIP_SSL_CERT_DOWNLOAD` work as the variable name suggest, that would also solve the problem and not require an additional variable.

**Comment by jackdelab** [comment]
@thrau Is this still the plan for supporting custom SSL certs on the non-pro localstack?

**Comment by thrau** [comment]
@jackdelab for community there should be nothing keeping you from using a custom cert. all you need to do is to set `SKIP_SSL_CERT_DOWNLOAD=1` and mount your pem file into the container to this location:  `/var/lib/localstack/.cache/server.test.pem`, and you should be good to go. `localstack:4566` accepts both HTTP and HTTPS traffic.

For Jordan's @jdelaune problem we have a fix in place that we will ship tomorrow!

**Comment by jackdelab** [comment]
@thrau I can confirm that placing the custom SSL cert in /var/lib/localstack/cache/server.test.pem is working for me on localstack version 0.13.3.3. Thanks again!
